.github/workflows/build.yaml

```diff
@@ -0,0 +1,49 @@
+name: Quick Go unit tests
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    env:
+      GOPATH: ${{ github.workspace }}
+      GO111MODULE: off
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        path: ./src/github.com/${{ github.repository }}
+    - name: Cache go pkg directory
+      id: cache-go-pkg
+      uses: actions/cache@v1
+      with:
+        path: ${{ github.workspace }}/pkg
+        key: ${{ runner.os }}-go-pkg
+    - name: Cache go vendor packages
+      id: cache-go-vendor
+      uses: actions/cache@v1
+      with:
+        path: ${{ github.workspace }}/src/github.com/${{ github.repository }}/vendor
+        key: ${{ runner.os }}-go-vendor
+    # XXX: is this the right version of go?
+    - name: Set up go 1.10
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.10
+      id: go  # XXX: what is this for?
+    - name: Install govendor
+      run: go get -u github.com/kardianos/govendor
```

Comment:
Should we cache this too?

Reply:
I'd not cache this until we understand how that works. The rest of the code reuses but overwrites the cache when we need to. I'd like for us to be able to "move" as govendor changes.

Reply:
Do you expect govendor to change much given that it is effectively obsolete in a Go Modules world?  Could you just do "apt-get install govendor" instead?